### PR TITLE
Adds Star Wars: The Deckbuilding Game

### PR DESCRIPTION
**Pull request by ISaumya1011**
Hi, I want to contribute to this Repository. I have added a new Board Game "Star Wars" in this ReadMe file. Please see if this addition can be merged against label HacktoberFest 2023. Thank you so much.

Saumya

Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS
- [x] read [this year's Hacktoberfest welcome issue](https://github.com/edm00se/awesome-board-games/issues/269)
- [x] promised not to `@` me asking for review (I will see the notifications already, no need to add noise)

Once your PR is created, it will trigger a build a couple of automated checks, including a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them. The others include a check on the submission limit and also a preview of the site with your changes; please check them out, they are tools tonhelp us both.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2704,6 +2704,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
+
+![Star Wars](https://upload.wikimedia.org/wikipedia/en/thumb/e/e2/Star_Wars_The_Deckbuilding_Game.webp/229px-Star_Wars_The_Deckbuilding_Game.webp.png)
+
+| Players | Min. Age | Time   |
+| ------: | -------: | -----: |
+|       2 |       10 | 30-45m |
```
Comment (edm00se) [inline readme.md:2715]: BGG cites 12 as the minimum age, they're usually right and I don't see it listed on the Wikipedia page.

File: readme.md [hunk 2]
```diff
@@ -2704,6 +2704,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 | ------: | -------: | -----: |
 |   1 - 4 |       10 | 30-45m |
 
+### [Star Wars](https://en.wikipedia.org/wiki/Star_Wars:_The_Deckbuilding_Game)
```
Comment (edm00se) [inline readme.md:2707]: Please update the title.

```suggestion
### [Star Wars: The Deckbuilding Game](https://en.wikipedia.org/wiki/Star_Wars:_The_Deckbuilding_Game) 
```

**Review by edm00se — CHANGES_REQUESTED** [review]
This is looking pretty good. Please update the minimum age in the stats block and the title.

**Comment by ISaumya1011** [comment]
> This is looking pretty good. Please update the minimum age in the stats block and the title.

Thank you. I've made the changes, kindly check them and please let me know if something else needs to be done.

File: readme.md (hunk 1 above)
Comment (edm00se) [reply readme.md:2715]:
```suggestion
|       2 |       12 | 30-45m |
```

**Review by edm00se — APPROVED** [review]
In the future, if you have to make changes to a Pull Reauest you’ve opened, do so in the same branch. There’s no need to create new duplicate PRs.

Thanks for contributing.

refs #269

**Comment by ISaumya1011** [comment]
Thank you so much for accepting and merging my Contribution. Being new to this community, I am still in the learning process.  Will take care of this in future. Thanks once again for guiding me the right path.

**Pull request merged**